src/Instance/ItemsList/ItemsList.js

```diff
@@ -247,7 +250,7 @@ const ItemsList = ({
       columnWidths={columnWidths}
       pagingType={MCLPagingTypes.PREV_NEXT}
       totalCount={total}
-      nonInteractiveHeaders={['loanType', 'effectiveLocation', 'materialType']}
+      nonInteractiveHeaders={draggable ? visibleColumns : ['order', 'loanType', 'effectiveLocation', 'materialType']}
```

Comment:
shouldn't be order sortable when non dragging?

Reply:
Added sorting by Order column